docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst

```diff
@@ -1,2 +1,238 @@
+
+***************************
+Dataset preparation
+***************************
+
+Download and prepare `VOC2012 dataset <http://host.robots.ox.ac.uk/pascal/VOC/voc2012>`_ with the following command:
+
+.. code-block::
+
+  cd data
+  wget http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar
+  tar -xvf VOCtrainval_11-May-2012.tar
+  cd ..
+
+.. image:: ../../../../../utils/images/voc_example.png
+  :width: 600
+
+The dataset contains a set of RGB images with 20 semantic labels such as aeroplane, bicycle, bird, car, person, etc. The images are stored in the following format:
+
+.. code-block::
+
+  VOC2012
+  ├── Annotations
+  ├── ImageSets
+  ├── JPEGImages
+  ├── SegmentationClass
+  ├── SegmentationObject
+
+
+
+*********
+Training
+*********
+
+1. First of all, we need to choose which semantic segmentation model will we train.
+The list of supported templates for semantic segmentation is available with the command line below.
+
+.. note::
+
+  The characteristics and detailed comparison of the models could be found in :doc:`Explanation section <../../../explanation/algorithms/segmentation/semantic_segmentation>`.
+
+  We also can modify the architecture of supported models with various backbones, please refer to the :doc:`advanced tutorial for model customization <../../advanced/backbones>`.
+
+.. code-block::
+
+  (otx) ...$ otx find --task segmentation
+  
+  +--------------+-----------------------------------------------------+--------------------+--------------------------------------------------------------------------+
+  |     TASK     |                          ID                         |        NAME        |                                BASE PATH                                 |
+  +--------------+-----------------------------------------------------+--------------------+--------------------------------------------------------------------------+
+  | SEGMENTATION |    Custom_Semantic_Segmentation_Lite-HRNet-18_OCR   |   Lite-HRNet-18    |   otx/algorithms/segmentation/configs/ocr_lite_hrnet_18/template.yaml    |
+  | SEGMENTATION | Custom_Semantic_Segmentation_Lite-HRNet-18-mod2_OCR | Lite-HRNet-18-mod2 | otx/algorithms/segmentation/configs/ocr_lite_hrnet_18_mod2/template.yaml |
+  | SEGMENTATION |  Custom_Semantic_Segmentation_Lite-HRNet-s-mod2_OCR | Lite-HRNet-s-mod2  | otx/algorithms/segmentation/configs/ocr_lite_hrnet_s_mod2/template.yaml  |
+  | SEGMENTATION |  Custom_Semantic_Segmentation_Lite-HRNet-x-mod3_OCR | Lite-HRNet-x-mod3  | otx/algorithms/segmentation/configs/ocr_lite_hrnet_x_mod3/template.yaml  |
+  +--------------+-----------------------------------------------------+--------------------+--------------------------------------------------------------------------+
+
+Note that we do not attach OCR head for supported models in default. We remain the suffix '_OCR' in ID just for backward compatibility.
```

Comment:
Emphasizing this sentence is more beneficial to users.

"Note that we do not attach OCR head for supported models in default."

Reply:
I made it NOTE!!